ansible/roles/tenderdash/tasks/main.yml

```diff
@@ -50,3 +50,32 @@
     state: present
     pull: always
     recreate: always
+
+- name: Configure logrotate for Tenderdash logs
+  ansible.builtin.copy:
+    dest: /etc/logrotate.d/tenderdash
+    content: |
+      {{ tenderdash_compose_path }}/tenderdash/logs/*.log {
+          daily
+          rotate 3
+          compress
+          delaycompress
+          missingok
+          notifempty
+          create 0644 {{ tenderdash_uid }} {{ tenderdash_gid }}
+      }
+    owner: '{{ tenderdash_uid }}'
+    group: '{{ tenderdash_gid }}'
+    mode: '0644'
+  when: tenderdash_network == 'testnet'
```

Comment:
I think we can use this config for all networks, given that logs are saved to elasticsearch.

Reply:
Done